packages/styles/src/typography/body.css

```diff
@@ -0,0 +1,42 @@
+/**
+ * Body text is used for long-form content where a paragraph or multiple lines of text are required. Body text styles are optimized for reading as a large chunk of information through line height and paragraph spacing. Inline links sit within body text styles and inherit the same font values. 
+ * UI text is text used in UI components and uses the same text stylings.
+ * The default font size is medium (16 px). Synergy supports three additional font sizes for body copy.
+ * It is also possible to format the text bold or semibold for certain text passages and UI elements such as label.
```

Comment:
I would like to stick to the convention we have in Shoelace where the component descriptions are SUPER short. This feels like information that should be in Storybook and therefore should be moved to `tokens/_docs`

Reply:
The contents are now all moved to docs tokens.